src/commands/zork.js

```diff
@@ -0,0 +1,44 @@
+const BaseCommand = require('../base-commands/base-command');
+
+class Zork extends BaseCommand {
+  constructor(argv, config, secureStorage) {
+    super(argv, config, secureStorage);
+    this.exec = require('await-exec');
+    this.findZork = () => require('zorkjs'); // eslint-disable-line node/no-extraneous-require,node/no-missing-require
```

Comment:
What does this comment do? Is this somehow issuing commands to the linter?

Reply:
Yeah... we don't have a direct dependency on the `zorkjs` module, but eslint tries to make sure you don't reference packages that aren't in your package.json. This is how you disable specific linter rules for a single line of code.